pkg/kubectl/rollout_status.go

```diff
@@ -98,8 +99,14 @@ func (s *DaemonSetStatusViewer) Status(namespace, name string, revision int64) (
 		if daemon.Status.UpdatedNumberScheduled < daemon.Status.DesiredNumberScheduled {
 			return fmt.Sprintf("Waiting for rollout to finish: %d out of %d new pod have been updated...\n", daemon.Status.UpdatedNumberScheduled, daemon.Status.DesiredNumberScheduled), false, nil
 		}
-		if daemon.Status.NumberAvailable < daemon.Status.DesiredNumberScheduled {
-			return fmt.Sprintf("Waiting for rollout to finish: %d of %d updated pods are available...\n", daemon.Status.NumberAvailable, daemon.Status.DesiredNumberScheduled), false, nil
+
+		var maxUnavailable int
+		if daemon.Spec.UpdateStrategy.Type == extensions.RollingUpdateDaemonSetStrategyType {
```

Comment:
Does it make sense for us to support DS in `rollout status` if the RollingUpdate strategy is not used?